### ISSUE TITLE
Model about Consumer to Shop

### ISSUE THREAD

**Issue by chenyingxue0124** [liuziwei7/fashion-landmarks#7]
Dear Ziwei Liu:
I have read your paper:DeepFashion: Powering Robust Clothes Recognition and Retrieval with Rich Annotations， and it is very helpful for my research.
You mentioned four benchmarks in your paper but I only found "fashion-landmarks" and "landmark detection" models in your GitHub.
Could you release the model about "Consumer to Shop" ? 
It will very helpful for my research.
Thank you very much.

**Comment by vivienzou1** [comment]
yes, can you release the pre-trained or code of "consumer to shop"?
